src/commonMain/kotlin/fr/acinq/lightning/wire/MessageOnion.kt

```diff
@@ -6,19 +6,20 @@ import fr.acinq.bitcoin.io.ByteArrayInput
 import fr.acinq.bitcoin.io.ByteArrayOutput
 import fr.acinq.bitcoin.io.Input
 import fr.acinq.bitcoin.io.Output
+import fr.acinq.lightning.NodeId
 import fr.acinq.lightning.crypto.RouteBlinding
 
-
-sealed class OnionMessagePayloadTlv : Tlv {
+sealed interface OnionMessagePayloadTlv : Tlv {
```

Comment:
Why did you change that to a `sealed interface`? For that type of cases (functionally, an `enum`), Kotlin usually recommends using a `sealed class`.

Reply:
I actually thought that sealed interface was the way to go. I'll use sealed class from now on.